k-distribution/tutorial/1_k/2_imp/lesson_5/imp.md

```diff
@@ -1,30 +1,30 @@
 // Copyright (c) 2014-2019 K Team. All Rights Reserved.
```

Comment:
If you view this file on GitHub you will see that this shows up at the top of the document. Please change this to a markdown comment.